### PR TITLE
Appends map scan link

### PR DESCRIPTION
**Pull request by cdmo**
#671 

Waiting on sample marc update but otherwise readay

### CODE REVIEW

**Comment by cdmo** [comment]
We could just add the new mrc data ourselves too. As in I can get the marc files from our catalog and just add them to the repo and modify the generation script (which still isn't working for me anyway)

**Comment by cdmo** [comment]
@banukutlu did you see my above question? Thoughts?

**Comment by banukutlu** [comment]
> @banukutlu did you see my above question? Thoughts?

yes, per our convo, I am 👍  for editing the script to get marc files from our catalog.

File: lib/tasks/blackcat.rake [hunk 1]
```diff
@@ -28,7 +28,7 @@ namespace :blackcat do
       # with examples that we are seeking and she will produce it and place it on the symphony server.
       traject_path = Rails.root.join('../psulib_traject')
       fixtures = Rails.root.join('spec/fixtures/current_fixtures.json')
-      marc_file = Rails.root.join('solr/sample_data/sample_psucat.mrc')
+      marc_file = Rails.root.join('solr/sample_data/*.mrc')
```
Comment (banukutlu) [inline lib/tasks/blackcat.rake:31]: 👌

**Review by banukutlu — APPROVED** [review]
👍

**Review by banukutlu — APPROVED** [review]

**Pull request merged**